### DISCUSSION TITLE
Adhesive for LED to Heatsink

### DISCUSSION THREAD

**Discussion by polygonfuture** [srozum/film_camera_tester#17] [Q&A] [answered]
Hi @srozum, Asked this in another thread but it may have gotten lost.

What type of adhesive / thermal tape did you use for mounting the LED to the heatsink?

**Comment by srozum** [comment, marked as answer]
This is most common thermal glue/plaster, but either would work.

![thermal_glue](https://user-images.githubusercontent.com/56254/226252251-cef14634-dd9f-4341-94ee-1613aca74eeb.jpg)
